### PR TITLE
adds tests for RouteRow Component

### PR DESCRIPTION
**Pull request by invincibleJai**
adds tests for RouteRow Component

Tracks: https://issues.redhat.com/browse/ODC-2641

### CODE REVIEW

File: frontend/packages/knative-plugin/src/components/routes/__tests__/RouteRow.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as _ from 'lodash';
+import { TableData } from '@console/internal/components/factory';
+import { ResourceLink, ExternalLink } from '@console/internal/components/utils';
+import { knativeRouteObj } from '@console/dev-console/src/components/topology/__tests__/topology-knative-test-data';
```
Comment (vikram-raj) [inline frontend/packages/knative-plugin/src/components/routes/__tests__/RouteRow.spec.tsx]: Same as this https://github.com/openshift/console/pull/3833/files#r361688840

**Comment by andrewballantyne** [comment]
/hold cancel
/retest
#3847 has fixed the e2e tests

**Pull request merged**